golang/pkg/signaling/server.go

```diff
@@ -13,6 +13,12 @@ var upgrader = websocket.Upgrader{
 	WriteBufferSize: 1024,
 }
 
+upgrader.CheckOrigin = func(r *http.Request) bool {
+  // do some kind of validation logic here
+  logrus.Debugf("Header contains origin %s", r.Header["Origin"]) // file://file:///Users/adam/Projects/adam-club/golang/static/index.html
```

Comment:
Something to note, `file://` will never work for local dev because requesting the video/mic requires TLS/SSL or the origin be localhost.

Reply:
It would be good to allow `localhost:*` or any origins in a list supplied by env var. Please make a low priority ticket to build this out. Tag it with a `1.0 blocker` tag too. Thanks.